src/include/mallocMC/creationPolicies/Scatter_impl.hpp

```diff
@@ -132,6 +132,8 @@ namespace ScatterKernelDetail{
       //static const uint32 minChunkSize0 = pagesize/(32*32);
       static const uint32 minChunkSize1 = 0x10;
       static const uint32 HierarchyThreshold =  (pagesize - 2*sizeof(uint32))/33;
+      static const uint32 tmp_maxOPM = minChunkSize1 > HierarchyThreshold ? 0 : pagesize/(32*minChunkSize1 + 1);
```

Comment:
Why `+1`?
Should it be a integer round up like `(pagesize+(32*minChunkSize1-1)/(32*minChunkSize1)`?